content/post/20200426-kubernetes-the-universal-control-plane/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Kubernetes: The Universal Control Plane"
+date: 2020-04-26T23:02:29-07:00
+tags:
+  - Kubernetes
+  - cloud
+---
+
+> The innovation is in the API
```

Comment:
I'd move this further down. Maybe the third section. It's a very punchy sentence and but upfront I have no context until at least 3 paragraphs in what you mean